ApollosProject/apollos-prototype
# Removes Caching of Auth Tokens

## DESCRIPTION

![Screen Shot 2019-10-15 at 1 36 28 PM](https://user-images.githubusercontent.com/2659478/66855207-d87cf200-ef50-11e9-96c8-865fd7c7eab9.png)


### What does this PR do, or why is it needed?

Just makes the auth token query `network-only`

### What design trade-offs/decisions were made?

None

### How do I test this PR?

Click the "InAppBrowser" link in the test panel multiple times, should log in every time.

### What automated tests did you write?

None

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
